reference/orders.v2.oas2.yml

```diff
@@ -4295,6 +4305,17 @@ components:
                 type: integer
               value:
                 type: string
+                description: |-
+                  Depending on the option type, value can be one of the following:
+                  - The variant option value id or the modifier value id for modifier types with a list of defined values, such as drop down or checkbox modifiers.
+                  - The modifier value for modifier types without a list of defined values, such as text field or date field modifiers.
+
+                  Notes:
+                  - The following option types are not currently supported via api:
+                    - File Upload
+                    - Pick List
+                  - For date modifiers use either the `ISO-8601` or the `YYYY-mm-dd` format. The date values are saved and retuned as timestamps. For values entered using the YYYY-mm-dd format the store timezone is used when creating the timestamp.
+                  - For mulit-line text field modifiers, use the `\n` characters for separating the lines.
```

Comment:
typo "mulit-line" 😄 

Also we may need to mention about those multi-line text modifiers only support 2 lines of text

Reply:
The two line constraint, is only when they have set it themselves in catalog.